snippets.py

```diff
@@ -198,7 +194,8 @@ def user_page_handler():
     if not _can_view_private_snippets(_current_user_email(), user_email):
         snippets = [snippet for snippet in snippets if not snippet.private]
     snippets = util.fill_in_missing_snippets(snippets, user,
-                                             user_email, _TODAY_FN())
+                                             user_email,
+                                             datetime.datetime.now())
```

Comment:
You may find it nicer to just set `now = datetime.datetime.now()` at the top of this function and use it everywhere.

Reply:
You're right, that is much nicer.